appveyor.yml

```diff
@@ -0,0 +1,23 @@
+
+os: Visual Studio 2017
+
+environment:
+  matrix:
+    - ARCH:                x86
+      BRANCH:              master
+
+cache:
+  - bin\lld-link.exe -> windows\build_lld.bat
+
+artifacts:
+  - path: bin\lld-link.exe
```

Comment:
Not sure whether you could use variables here `bin\lld-link-%VERSION%.zip`.

Reply:
According to the docs `lld-link-$(VERSION).zip` should work.